ballerina/client.bal

```diff
@@ -256,6 +256,61 @@ public isolated client class Client {
         http:Response httpResponse = check self.amazonS3->delete(requestURI, request);
         return handleHttpResponse(httpResponse);
     }
+    
+    # Generates a presigned URL for the object.
+    #
+    # + bucketName - The name of the bucket  
+    # + objectName - The name of the object  
+    # + expirationTime - The time until the presigned URL is valid in seconds
+    # + httpMethod - The HTTP method to be used, either GET or PUT  
+    # + partNo - The part number of the object, when uploading multipart objects
+    # + uploadId - The upload ID of the multipart upload
+    # + return - If success, a presigned URL, else an error
+    @display {label: "Create Presigned URL"}
+    remote isolated function createPresignedURL(
+            @display {label: "Bucket Name"} string bucketName,
+            @display {label: "Object Name"} string objectName,
+            @display {label: "HTTP Method"} string httpMethod,
+            @display {label: "Expiration Time"} int expirationTime = 3600,
+            @display {label: "Part Number"} int partNo = 0,
+            @display {label: "Upload ID"} string? uploadId = ())
+        returns @tainted string|error {
+
+        [string, string] [amzDateStr, shortDateStr] = check generateDateString();
+
+        map<string> queryParams = {
+            [X_AMZ_ALGORITHM] : AWS4_HMAC_SHA256,
+            [X_AMZ_CREDENTIAL] : string `${self.accessKeyId}/${shortDateStr}/${self.region}/${SERVICE_NAME}/
+                ${TERMINATION_STRING}`,
```

Comment:
This will a line break after / and before the interpolation. Break within the interpolation instead.